facebook/delphiFacebook/R/contingency_tables.R

```diff
@@ -0,0 +1,1068 @@
+    "b_consent" = "S1",
+    "b_hh_fever" = "hh_fever", # A1_1
+    "b_hh_sore_throat" = "hh_soar_throat", # A1_2
+    "b_hh_cough" = "hh_cough", # A1_3
+    "b_hh_shortness_of_breath" = "hh_short_breath", # A1_4
+    "b_hh_difficulty_breathing" = "hh_diff_breath", # A1_5
+    "b_tested_ever" = "B8",
+    "b_tested_14d" = "t_tested_14d", # B10; "No" coded as 3, but dealt with in conversion to "t_tested_14d"
+    "b_wanted_test_14d" = "t_wanted_test_14d", # B12
+    "b_state_travel" = "C6", # c_travel_state
+    "b_contact_tested_pos" = "C11",
+    "b_contact_tested_pos_hh" = "C12",
+    "b_hispanic" = "D6",
+    "b_worked_4w" = "D9",
+    "b_worked_outside_home_4w" = "D10",
+    "b_took_temp" = "B3",
+    "b_flu_shot_12m" = "C2",
+    "b_worked_outside_home_5d" = "c_work_outside_5d", # C3
+    "b_worked_healthcare_5d" = "C4",
+    "b_worked_nursing_home_5d" = "C5",
+    "b_anxious" = "mh_anxious", # Binary version of C8_1
+    "b_depressed" = "mh_depressed", # Binary version of C8_2
+    "b_isolated" = "mh_isolated", # Binary version of C8_3
+    "b_worried_family_ill" = "mh_worried_ill", # Binary version of C9   
+    "b_public_mask_often" = "c_mask_often", # Binary version of C14
+    "b_tested_pos_14d" = "t_tested_positive_14d", # B10a; binary with an "I don't know" (3) option
+    "b_tested_pos_ever" = "B11", # binary with an "I don't know" (3) option
+    "b_have_cli" = "is_cli", # Based on symptoms in A1
+    "b_have_ili" = "is_ili", # Based on symptoms in A1
+    "b_cmnty_have_cli" = "community_yes",
+    "b_hh_or_cmnty_have_cli" = "hh_community_yes",
+    
+    ## multiple choice (mc)
+    # Can only select one of n > 2 choices
+    "mc_state" = "A3b",
+    "mc_mask_often" = "C14",
+    "mc_anxiety" = "C8_1",
+    "mc_depression" = "C8_2",
+    "mc_isolation" = "C8_3",
+    "mc_worried_family_ill" = "C9",
+    "mc_financial_worry" = "C15",
+    "mc_gender" = "D1",
+    "mc_age" = "D2",
+    "mc_race" = "D7",
+    "mc_education" = "D8",
+    "mc_occupational_group" = "Q64",
+    "mc_job_type_cmnty_social" = "Q65",
+    "mc_job_type_education" = "Q66",
+    "mc_job_type_arts_media" = "Q67",
+    "mc_job_type_healthcare" = "Q68",
+    "mc_job_type_healthcare_support" = "Q69",
+    "mc_job_type_protective" = "Q70",
+    "mc_job_type_food" = "Q71",
+    "mc_job_type_maintenance" = "Q72",
+    "mc_job_type_personal_care" = "Q73",
+    "mc_job_type_sales" = "Q74",
+    "mc_job_type_office_admin" = "Q75",
+    "mc_job_type_construction" = "Q76",
+    "mc_job_type_repair" = "Q77",
+    "mc_job_type_production" = "Q78",
+    "mc_job_type_transport" = "Q79",
+    "mc_occupational_group_other" = "Q80",
+    "mc_cough_mucus" = "B4",
+    "mc_tested_current_illness" = "B5",
+    "mc_hospital" = "B6",
+    "mc_social_avoidance" = "C7",
+    "mc_financial_threat" = "Q36",
+    "mc_cmnty_mask_prevalence" = "C16",
+    "mc_flu_shot_jun2020" = "C17",
+    "mc_children_grade" = "E1",
+    "mc_children_school" = "E2",
```

Comment:
I don't think E1 and E2 are coded this way. See 1970207822. There's one response per row in the matrix of choices, and each of those is multiple choice, so you'd separately have `E1_1`, `E1_2`, etc.